boards/shields/adafruit_aw9523/adafruit_aw9523.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This shield has the STEMMA-QT/Qwiic connector, but devicetree defines it as
+ * an electrically compatible Grove connector for convenience.
```

Comment:
I appreciate you taking the time to document this "convenience shortcut" you took but I think you would actually need to define a proper connector + conventional aliases for Qwiic since otherwise this will be problematic for people trying to use this shield on board that has both types of connectors IMO